python/taichi/lang/snode.py

```diff
@@ -1,7 +1,13 @@
 import numbers
 
+# The reason we import just the taichi.core.util module, instead of the ti_core
+# object within it, is that ti_core is stateful. While in practice ti_core is
+# loaded during the import procedure, it's probably still good to delay the
+# access to it.
+from taichi.core import util as cutil
```

Comment:
We should probably do this for all the other aliases of `ti_core` as well?

Reply:
Sounds good :-)